qcodes/instrument_drivers/Aviosys/IP_Power_9258S.py

```diff
@@ -0,0 +1,92 @@
+from qcodes import Instrument, InstrumentChannel, ChannelList
+from qcodes.utils.validators import Enum
+from qcodes.utils.helpers import create_on_off_val_mapping
+import urllib
+
+
+class PowerChannel(InstrumentChannel):
+    """
+    Channel class for a socket on the Aviosys IP Power 9258S.
+    Args:
+        parent (Instrument): Parent instrument.
+        name (str): Channel name.
+        channel (str): Alphabetic channel id.
+    Attributes:
+        channel (str): Alphabetic channel id.
+        channel_id (int): Numeric channel id.
+    """
+
+    _channel_values = Enum('A', 'B', 'C', 'D')
+    _channel_ids = {'A': 1, 'B': 2, 'C': 3, 'D': 4}
+
+    def __init__(self, parent, name, channel):
+
+        super().__init__(parent, name)
+
+        # validate the channel id
+        self._channel_values.validate(channel)
+        self.channel = channel
+        self.channel_id = self._channel_ids[channel]
+
+        # add parameters
+        self.add_parameter('power',
```

Comment:
would you agree that `enabled` or `power_enabled` is a better name instead of `power` because it is more specific and aligns well with the boolean True/False values that the parameter has? Also, in the example notebook, you already invite the users to use True/False for setting `power` parameter as opposed to setting it to 'on' or 'off', which makes me think that `enabled` or `power_enabled` is really the name of the parameter you were looking for.

If so, could you rename it to `power_enabled`/`enabled`?